workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -265,55 +171,29 @@ public bool HasComponent<T>(Entity entity)
 
         public bool HasComponent(Entity entity, ComponentType componentType)
         {
-            return EntityManager.HasComponent(entity, componentType);
+            return entityManager.HasComponent(entity, componentType);
         }
 
-        public void CreateEntity(long entityId)
+        public bool TryGetEntity(long entityId, out Entity entity)
         {
-            if (entityMapping.ContainsKey(entityId))
-            {
-                Debug.LogErrorFormat(Errors.AddEntityButEntityIdAlreadyExists, entityId);
-                return;
-            }
-
-            var entity = EntityManager.CreateEntity();
-            EntityManager.AddComponentData(entity, new SpatialEntityId
-            {
-                EntityId = entityId
-            });
-            EntityManager.AddComponentData(entity, new NewlyAddedSpatialOSEntity());
-
-            AddAllCommandRequestSenders(entity, entityId);
-            entityMapping.Add(entityId, entity);
+            return entityMapping.TryGetValue(entityId, out entity);
         }
 
         public void AddGameObjectEntity(Entity entity, GameObject gameObject)
         {
             gameObjectManager.AddGameObjectEntity(entity, gameObject);
         }
 
-        public void RemoveEntity(long entityId)
+        public void HandleAuthorityChange<T>(long entityId, Authority authority,
+            ComponentPool<AuthoritiesChanged<T>> pool)
         {
             Entity entity;
-            if (!entityMapping.TryGetValue(entityId, out entity))
+            if (!TryGetEntity(entityId, out entity))
             {
-                Debug.LogErrorFormat(Errors.DeleteNonExistantEntity, entityId);
+                Debug.LogErrorFormat(Errors.NoCorrespondingEntityForEntityId, typeof(T).Name, entityId);
```

Comment:
Can you use `nameof(T)` here?

Reply:
I think its just run time vs compile time, so yes probably? Not sure if worth making the change tbh